src/schema/uuid/thread/types.ts

```diff
@@ -43,11 +58,10 @@ export interface ThreadData {
 
 export interface ThreadResolvers {
   Thread: {
+    id: Resolver<ThreadData, never, string | null>
```

Comment:
```suggestion
    id: Resolver<ThreadData, never, string>
```

Property is `id: String!`. Thus it needs to return a string.